templates/default/papertrail.conf.erb

```diff
@@ -20,12 +20,9 @@ $ActionQueueSaveOnShutdown on
 $ActionQueueTimeoutEnqueue 10
 $ActionQueueDiscardSeverity 0
 
-<%- if @fixhostname %>
-
-*.*   @@<%= @host %>:<%= @port %>;FixHostname
-
-<%- else %>
-
-*.*   @@<%= @host %>:<%= @port %>
-
-<%- end %>
+<%   case node['papertrail']['protocol'] -%>
```

Comment:
i've remove the protocol stuff from this branch. honestly i'm not that deep into syslog ;) i just picked the `@@` vs. `@` from the papertrail guides and it worked surprisingly well.

so the Action* directives should be adjusted or even removed completely?